rdwatch/scoring/views/model_run.py

```diff
@@ -507,11 +507,9 @@ def list_model_runs(
 @router.get('/{id}/', response={200: ModelRunDetailSchema})
 def get_model_run(request: HttpRequest, id: UUID4):
     try:
-        EvaluationRun.objects.get(pk=id)
-        data = get_queryset()
+        data = get_object_or_404(get_queryset(), id=id)
```

Comment:
I don't think `get_object_or_404` raises `EvaluationRun.DoesNotExist` here, so wouldn't this fail prematurely?

Reply:
Thanks for finding that, you're correct.  This won't properly swap to the proposal query.

Reply:
Decided to give a more descriptive error with missing EvaluationRun/AnnotationProposalSet.